JavaBytecodeCompiler/src/main/java/org/openzen/zenscript/javabytecode/compiler/JavaSwitchKeyVariableVisitor.java

```diff
@@ -62,8 +62,10 @@ public Void acceptVariantOption(VariantOptionSwitchValue key) {
 			javaWriter.dup();
 			javaWriter.getField(
 					javaVariantOption.variantOptionClass.internalName,
-					"field" + fieldNumber++,
-					context.getDescriptor(binding.type));
+					"field" + fieldNumber,
```

Comment:
💭 At a later time we might want to check whether we want some of Silk's Name-Mangling Magic here, or if `field0` are fine?